actions.yml

```
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  # Triggers the workflow on push or pull request events
+  push:
+    branches: [ master ]
+  pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Install dependencies and build latest.zip
```

Isn't a `setup-node` step needed here? (from the [example](https://docs.github.com/pt/free-pro-team@latest/actions/guides/building-and-testing-nodejs) in the docs)

```suggestion
      - name: Install dependencies and build latest.zip
        uses: actions/setup-node@v1
        with:
          node-version: ${{ matrix.node-version }}
```